mbs_results/data_cleaning.py

```diff
@@ -182,14 +182,63 @@ def load_manual_constructions(
     )
 
 
+def join_manual_constructions(
+    df: pd.DataFrame,
+    manual_constructions: pd.DataFrame,
+    reference: str,
+    period: str,
+    **config
+):
+    """
+    function to change datatypes of columns based on config file
+
+    Parameters
+    ----------
+    df: pd.DataFrame
+        dataframe with combined responses and contributors columns
+        index is expected to be 'period' and 'reference'
+    manual_constructions_path: str
+        the manual construction dataframe
+    reference: str
+        the name of the reference column
+    period: str
+        the name of the period column
+    **config: Dict
+        main pipeline configuration. Can be used to input the entire config dictionary
+
+    Returns
+    -------
+    pd.DataFrame
+        dataframe with correctly formatted column datatypes.
+    """
+    if manual_constructions[period].dtype != df[period].dtype:
```

Comment:
Loved this line, perhaps having it as a utility function, i think it's very useful functionality something like `is_same_dtype`  and return a boolean

Reply:
Refactored to add this as new function in data_cleaning module (can move if needed)